webpack.config.js

```diff
@@ -57,7 +57,7 @@ Encore
     .addAliases({
         '@': path.resolve(__dirname, 'assets', 'js'),
         'images': path.resolve(__dirname, 'assets', 'images'),
-        vue: 'vue/dist/vue.js'
+        vue: 'vue/dist/vue.min.js'
```

Comment:
This seems to make it work.
See:  https://github.com/symfony/webpack-encore/issues/625
```suggestion
        vue: Encore.isProduction() ? 'vue/dist/vue.min.js': 'vue/dist/vue.js'
```